portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACWorkspace.macro

```diff
@@ -108,6 +108,47 @@ definition {
 		ACUtils.dismissOnboardingModal();
 	}
 
+	macro setFriendlyURL {
```

Comment:
Hey @calvinctvo, I think it would be a good idea to put it this way at the beginning of your macro:

`var key_friendlyURL = ${friendlyURL};`

That way it is clearer to the person looking at the code that you passed the value of the variable `${friendlyURL}` in when you declared this macro in your test. That way the person will understand that the value of this variable is being placed in the variable `${key_friendlyURL}` that will be used in your path.

The way you did it is correct, but I think these changes make your code clearer. Did you understand what I wanted to say to you? If it's not clear I can explain again

Reply:
Hey @ViniLopes20 , yes it'd be less confusing. I was just used to passing in a variable without `key_` so I forgot the rule that we had on passing variable in path. Thanks for catching that!